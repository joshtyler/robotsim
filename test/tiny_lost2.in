0 1
0 0 N
FF
0 0 N
RRF
