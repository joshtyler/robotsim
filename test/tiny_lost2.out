0 1 N LOST
0 0 S LOST
